prow/plugins/mergecommitblock/mergecommitblock.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package mergecommitblock adds a do-not-merge label to pull requests which contain merge commits
+// Merge commits are defined as commits that contain more than one parent commit SHA
+
+package mergecommitblock
+
+const pluginName = "block-mc"
```

Comment:
This is going to be user-facing, so I think we should prefer `mergecommitblocker` or similar.